### ISSUE TITLE
Allow plasmoid to be fixed size

### ISSUE THREAD

**Issue by protist** [antroids/application-title-bar#1]
Thank you so much for this plasmoid! It works very well in Plasma 6.

However, the `Size mode` = `Fixed size` doesn't seem to work for me. I would have expected the width of the widget to stay constant, regardless of how much text is in it. However, it doesn't appear to be fixed. This is quite distracting, because when the title changes, the size does too, and all of my widgets to the right of this widget move around.

**Comment by antroids** [comment]
This option was for font size, I renamed it in new version.
I also added minimum/maximum size for Window title.
You can make width fixed by set the same values, since the title is the only element with variable width.

Fixed in 0.2.3

**Comment by protist** [comment]
Thanks for the quick fix. I *think* I've updated, but is the `About` tab still showing `Application Title Bar 0.1`?

I'm not sure that it's working for me though. I tried setting `Minimum width` and `Maximum width` both to 400. However, the size is still changing. I also tried reducing `Minimum width` to 399, but that still fails. Is there another setting I need to toggle? I tried to change `Font fit` to `Fixed size` or `Horizontal fit`, but it still wasn't working. I'm happy to provide more information if helpful.
